giza_actions/model.py

```diff
@@ -189,21 +197,28 @@ def _download_model(self, output_path: str):
                 f"Model version status is not completed {self.version.status}"
             )
 
-        onnx_model = self.version_client.download_original(
-            self.model.id, self.version.version
-        )
+        cache_str = f"{self.model.id}_{self.version.version}_model"
```

Comment:
As in the previous comment, let's remove this and use self._output_path so cache keys are more consistent